sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -716,4 +717,118 @@ public HttpPipeline getHttpPipeline() {
 
         return protocolLayer.getDigitalTwinModels().updateWithResponseAsync(modelId, updateOperation, context);
     }
+
+    //==================================================================================================================================================
+    // Component APIs
+    //==================================================================================================================================================
+
+    /**
+     * Get a component of a digital twin.
+     * @param digitalTwinId The Id of the digital twin to get the component from.
+     * @param componentPath The path of the component on the digital twin to retrieve.
+     * @return The application/json string representing the component of the digital twin.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<String> getComponent(String digitalTwinId, String componentPath) {
+        return getComponentWithResponse(digitalTwinId, componentPath)
+            .flatMap(response -> Mono.justOrEmpty(response.getValue()));
```

Comment:
can be a map instead

Reply:
```java
.map(DigitalTwinsResponse::getValue);
```

Reply:
So are these two basically the same then?

```java
.map(response -> response.getValue());
```

```java
.flatMap(response -> Mono.justOrEmpty(response.getValue()));
```

Reply:
Yes, they are the same.
The first one will take the response emitted by the previous Mono, transform it to `.getValue()` and emit the result.
The second one will take the response emitted by the previous Mono, transform it into `.getValue()`, put it into a Mono, flatten it and emit the result.

Reply:
So the end result from both of these transformed APIs will be the `.getValue()` items directly.